tools/bundle-builder/lib/depUtils.js

```diff
@@ -14,38 +14,39 @@ const fsp = require("fs").promises;
 const path = require("path");
 const depSolver = require("dependency-solver");
 
-/*
-  Given a package path, get its dependencies
-
-  @param {string} packages - package directory
-
-  @return {Object} An object mapping the package name to its dependencies, or null if no dependencies
-*/
+/**
+ * Given a package path, get its dependencies
+ * @param {string} packages - package directory
+ * @return {Object} An object mapping the package name to its dependencies, or null if no dependencies
+ */
 async function getDependencies(package) {
-	let pkg = JSON.parse(await fsp.readFile(path.join(package, "package.json")));
-	let dependencies = [];
-
-	if (pkg.devDependencies) {
-		dependencies = Object.keys(pkg.devDependencies).filter((dep) => {
-			return (
-				dep.indexOf("@spectrum-css") === 0 &&
-				dep !== "@spectrum-css/bundle-builder" &&
-				dep !== "@spectrum-css/component-builder" &&
-				dep !== "@spectrum-css/component-builder-simple"
-			);
-		});
-	}
-
-	return { name: pkg.name, dependencies: dependencies };
+	const {
+		name,
+		peerDependencies = {},
+		dependencies = {},
+		devDependencies = {}
+	} = await fsp.readFile(path.join(package, "package.json")).then(JSON.parse);
+
+	return {
+		name,
+		dependencies: [...new Set([
```

Comment:
This is the main part of the update - merges all dependencies and dedupes them for peer, dev, and deps to capture dependencies in the new package structure.